crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -78,7 +78,7 @@ jQuery(document).ready(function($) {
     ]
   });
 
-  $('[data-blockui]').live('submit', function(event) {
+  $('[data-blockui]').live('click submit', function(event) {
```

Comment:
This will cause problems see https://github.com/crowbar/barclamp-crowbar/pull/1380